modules/cms-kit/src/Volo.CmsKit.Admin.Application/Volo/CmsKit/Admin/CmsKitAdminApplicationModule.cs

```diff
@@ -44,11 +46,35 @@ private void ConfigureTagOptions()
                         new TagEntityTypeDefiniton(
                             BlogPostConsts.EntityType,
                             LocalizableString.Create<CmsKitResource>("BlogPost"),
-                            CmsKitAdminPermissions.BlogPosts.Update,
-                            CmsKitAdminPermissions.BlogPosts.Update,
-                            CmsKitAdminPermissions.BlogPosts.Update));
+                            new[] { CmsKitAdminPermissions.BlogPosts.Create, CmsKitAdminPermissions.BlogPosts.Update },
+                            new[] { CmsKitAdminPermissions.BlogPosts.Create, CmsKitAdminPermissions.BlogPosts.Update },
+                            new[] { CmsKitAdminPermissions.BlogPosts.Create, CmsKitAdminPermissions.BlogPosts.Update }));
                 }
             });
+
+            if (GlobalFeatureManager.Instance.IsEnabled<MediaFeature>())
+            {
+                Configure<CmsKitMediaOptions>(options =>
+                {
+                    if (GlobalFeatureManager.Instance.IsEnabled<BlogsFeature>())
+                    {
+                        options.EntityTypes.AddIfNotContains(
+                            new MediaDescriptorDefinition(
+                                BlogPostConsts.EntityType,
+                                createPolicies: new[] { CmsKitAdminPermissions.BlogPosts.Create, CmsKitAdminPermissions.BlogPosts.Update },
+                                deletePolicies: new[] { CmsKitAdminPermissions.BlogPosts.Delete }));
+                    }
+
+                    if (GlobalFeatureManager.Instance.IsEnabled<PagesFeature>())
+                    {
+                        options.EntityTypes.AddIfNotContains(
+                            new MediaDescriptorDefinition(
+                                PageConsts.EntityType,
+                                createPolicies: new[] { CmsKitAdminPermissions.Pages.Create, CmsKitAdminPermissions.Pages.Update },
+                                deletePolicies: new[] { CmsKitAdminPermissions.Pages.Delete }));
```

Comment:
I think the user who has media create permission can delete the media as well.

Reply:
Can you check Commit b7217ab